charts/lm-container/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: lm-container
 description: A Helm chart for Logicmonitor's Kubernetes monitoring solutions
 type: application
-version: 6.2.2-rt01
+version: 7.0.0-rc02
```

Comment:
lm-container-7.0.0-rc01 was already published and hence bumping the version to 7.0.0-rc02.

Reference PR : https://github.com/logicmonitor/helm-charts-qa/pull/327/files#diff-829ac19bc7d7032b2e9dd5dc2a7f26a46de766b3e54bdcb04de2fea6f3733bf1R5